app/controllers/charities_controller.rb

```diff
@@ -7,6 +7,7 @@ def index
   def show
     @user = User.new
     @charity = Charity.find(params[:id])
+    @booking = Booking.find(params[:id])
```

Comment:
I'm not sure if this variable is ever used?

Reply:
Same! Thanks!